tabbycat/settings/heroku.py

```diff
@@ -124,6 +126,7 @@
             LoggingIntegration(event_level=logging.WARNING),
         ],
         send_default_pii=True,
+        release=TABBYCAT_VERSION
```

Comment:
Being picky, can we put a comma after this? My convention is to put trailing commas when the closing bracket is on a new line, that way if you want to add a new argument later (as you have here) or rearrange lines, it doesn't affect the other lines.